src/styles/createZIndex.ts

```diff
@@ -0,0 +1,39 @@
+import muiZIndex from '@mui/material/styles/zIndex';
```

Comment:
I think we can get rid of this import since we have the same import below. 

```suggestion
```

Reply:
For the rest of the team's understanding - Haley and I met and talked about this import (it was also causing issues in the tests). For now, we've decided to copy the zIndex values from MUI because 

1. We timeboxed trying to get the import to work with Jest
2. This will allow us to keep MUI's values at this point in time, even if they update in the future, and avoid breaking changes in zIndex

If it ever makes sense to, we can revisit this issue in the future